flang/lib/Lower/OpenMP.cpp

```diff
@@ -2178,7 +2178,7 @@ createAndSetPrivatizedLoopVar(Fortran::lower::AbstractConverter &converter,
 template <typename Op>
 static void createBodyOfOp(
     Op &op, Fortran::lower::AbstractConverter &converter, mlir::Location &loc,
-    Fortran::lower::pft::Evaluation &eval,
+    Fortran::lower::pft::Evaluation &eval, bool genNested,
```

Comment:
Nit: Add parameter description for `genNested`.

Reply:
Also, would `isCombined` be a better description, since that looks like the only place when it's going to be true? That way, it can be removed as a parameter to all `genSomeOp` functions of operations that can't appear in combined constructs as well.

Reply:
I prefer `genNested`, since I think of it as a directive to the callee: either lower the nested evaluations or not.

Reply:
I guess this is just a matter of taste, so feel free to leave it as is. The reason why I suggested the change is that there are two boolean flags passed in: `genNested` and `outerCombined`. The first one indicates _what_ it will do and the second indicates _when_ the caller should set it to true, so I thought it would be good if both conveyed information in the same way.

Reply:
I see what you mean.  Yeah, one flag is a directive, the other one is an information (the `outerCombined` is there to indicate whether data-sharing (privatization etc.) has already been initiated or not).  Hopefully the `genNested` will be eliminated eventually.